packages/calcite-components/src/components/input-message/input-message.scss

```diff
@@ -44,21 +31,20 @@
   color: var(--calcite-color-brand);
 }
 
-// Validation Text
-:host([status]) {
-  @apply text-color-1;
-}
-
-:host([status][scale="s"]) {
+:host([scale="s"]) {
```

Comment:
Yeah I'm not sure, maybe a question for design? What should be the spacing for the small scale @SkyeSeitz, @ashetland?

Reply:
In Figma, all three scales of Input Message have a 4px gap between the icon and text and 4px top padding.

Reply:
@ashetland should this be a ‘—calcite-internal-‘ prop? Just want to make sure we aren’t encouraging changing this if we don’t need to.

Reply:
`--calcite-input-message-spacing-value` is [documented](https://developers.arcgis.com/calcite-design-system/components/input-message/#styles), so we can't change it if this was what you were suggesting.

Reply:
Wait what are we changing? 🤣 I think I've misunderstood the ask.

Reply:
Can we deprecate it and remove in a future breaking change? 

I don't understand why we'd want to be customized - a user can already add spacing to the component when used standalone.

Within a Label - they can add space to their control, and in the future when Label is baked in to form elements, all this would do is allow the space between the control and the input message to be adjusted.

Reply:
> Can we deprecate it and remove in a future breaking change?
> 
> I don't understand why we'd want to be customized - a user can already add spacing to the component when used standalone.
> 
> Within a Label - they can add space to their control, and in the future when Label is baked in to form elements, all this would do is allow the space between the control and the input message to be adjusted.

I think we should keep `--calcite-input-message-spacing-value` because it's the only way to modify the spacing for the validation messages I'm adding in #8574.

Although I wouldn't be opposed to deprecating it and renaming it to something like `--calcite-input-message-margin-top` for clarity and consistency (e.g. compared to `--calcite-label-margin-bottom`).

Reply:
Modify by us or an end user? Maybe it's an `-internal` name if the former? 

I don't think it's valid for a user to change the spacing between Input Message we render and the component, just to change the spacing. This would only make sense IMO when they add other content, which itself could handle spacing. Just seems weird to have a property to space around a component - I know we have for Label but we don't want to in the future, that should be controlled by a parent group that spaces form controls.

Reply:
Yeah that makes sense, I didn't realize we were moving away from the label spacing CSS variable. We should probably do them both at the same time in that case. 

It looks like the only place `--calcite-input-message-spacing-value` is used internally will be in radio-button-group after #8561 is installed. So I could try to find a different way to do that, or we could change it to `-internal` like you mentioned.

Reply:
Probably fine for now, we can address when the form-components have Label functionality built in and that component is deprecated down the line.